packages/data-stores/src/plans/hooks/use-pricing-meta-for-grid-plans.ts

```diff
@@ -77,7 +77,7 @@ const usePricingMetaForGridPlans = ( {
 		purchaseId: currentPlan?.purchaseId,
 	} );
 	const selectedStorageOptions = useSelect(
-		( select ) => select( WpcomPlansUI.store ).getSelectedStorageOptions(),
+		( select ) => select( WpcomPlansUI.store ).getSelectedStorageOptions( selectedSiteId ?? 0 ),
```

Comment:
TODO: Consider empty state for siteId ( Ex. onboarding for new site )

Reply:
It looks like 0 is used to represent a non-existent siteId throughout the codebase. I believe it makes sense here as well, but feel free to chime in if it doesn't

Reply:
IIRC typescript gets upset at us for attempting to index an object with a `null` or `undefined` value. I'll double check tomorrow though